jsk_arc2017_baxter/euslisp/lib/pick-interface.l

```diff
@@ -248,6 +252,7 @@
           (ros::ros-info-green "[main] arm ~a: place object ~a in cardboard ~a" arm target-obj target-cardboard)
           (send self :update-json target-obj
                 :src (cons :bin target-bin) :dst (cons :cardboard target-cardboard))
+          (send self :add-finished-object arm target-obj)
```

Comment:
Do we need `(send self :add-postponed-object arm target-obj)` in https://github.com/start-jsk/jsk_apc/pull/2296/files#diff-6a34c6b505e3c7331c663f31dbd8145bR247 ?